app/cards/renaissance/treasurer.js

```diff
@@ -0,0 +1,139 @@
+Treasurer = class Treasurer extends Card {
+
+    types() {
+        return ['action']
+    }
+
+    coin_cost() {
+        return 5
+    }
+
+    static artifact_name() {
+        return 'Key'
+    }
+
+    static get_artifact_instance() {
+        return new Key()
+    }
+
+    play(game, player_cards) {
+        let gained_coins = CoinGainer.gain(game, player_cards, 3)
+        game.log.push(`&nbsp;&nbsp;<strong>${player_cards.username}</strong> gets +$${gained_coins}`)
+
+        let turn_event_id = TurnEventModel.insert({
+            game_id: game._id,
+            player_id: player_cards.player_id,
+            username: player_cards.username,
+            type: 'choose_options',
+            instructions: `Choose One:`,
+            minimum: 1,
+            maximum: 1,
+            options: [
+                { text: 'Trash Treasures from hand', value: 'trash' },
```

Comment:
Super minor, but I think the language here should read "Trash a treasure from hand" since you only get to trash a singular treasure